src/Symfony/Component/Console/Command/Command.php

```diff
@@ -447,6 +448,26 @@ public function getName()
     }
 
     /**
+     * @param bool $public Whether the command should be publicly shown or not.
+     * 
+     * @return Command The current instance
+     */
+    public function setPublic($public)
```

Comment:
we could meet in the middle at `setPublicity` as it's not favoring one way or another :)